### PR TITLE
Update for release 2.1.8

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: pages/download.html [hunk 1]
```diff
@@ -40,19 +40,19 @@
...
-previous_ubuntu32: https://downloads.mixxx.org/mixxx-2.1.7/mixxx-2.1.7-bionic-i386.deb
-previous_ubuntu64: https://downloads.mixxx.org/mixxx-2.1.7/mixxx-2.1.7-bionic-amd64.deb
-previous_linuxsrc: https://github.com/mixxxdj/mixxx/archive/release-2.1.7.tar.gz6
-previous_win32_analytics_conversion: /downloads/2.1.7-win32
-previous_win64_analytics_conversion: /downloads/2.1.7-win64
-previous_osxintel_analytics_conversion: /downloads/2.1.7-osxintel
-previous_linuxsrc_analytics_conversion: /downloads/2.1.7-linuxsrc
+previous_ubuntu32: https://downloads.mixxx.org/mixxx-2.1.8/mixxx-2.1.8-bionic-i386.deb
+previous_ubuntu64: https://downloads.mixxx.org/mixxx-2.1.8/mixxx-2.1.8-bionic-amd64.deb
+previous_linuxsrc: https://github.com/mixxxdj/mixxx/archive/release-2.1.8.tar.gz6
```
Comment (uklotzde) [inline pages/download.html]: Wrong suffix: ".gz6" instead of ".gz"
The link was already broken for 2.1.7

**Comment by uklotzde** [comment]
Wrong download link (inherited from 2.1.7), otherwise LGTM.

**Comment by daschuer** [comment]
Thank you.

**Comment by ywwg** [comment]
Should I announce on facebook / twitter?

**Comment by daschuer** [comment]
Yes that would be nice.
You may highlight that this release is recommended for those who have GUI leak issues with 2.2.0 on Mac-Os or Wayland and for OSs like Ubuntu Trust or Windows XP

**Pull request merged**